### PR TITLE
move all images into images/ directory

### PR DESCRIPTION
**Pull request by hbagdi**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

Moves images into a single `images/` directory.

**Which issue(s) this PR fixes**:

None

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/simple-gateway/index.html [hunk 1]
```diff
@@ -549,7 +549,7 @@ <h1 id="deploying-a-simple-gateway">Deploying a simple Gateway</h1>
 deployed together by the same owner. This represents a similar kind of model
 used for Ingress. In this guide, a Gateway and HTTPRoute are deployed which
 match all HTTP traffic and directs it to a single Service named <code>foo-svc</code>. </p>
-<p><img alt="Simple Gateway" src="../single-service-gateway.png" /></p>
+<p><img alt="Simple Gateway" src="single-service-gateway.png" /></p>
```
Comment (robscott) [inline docs/simple-gateway/index.html:552]: This doesn't seem right?

Comment (hbagdi) [reply docs/simple-gateway/index.html:552]: The images are inside `docs` directory in the generated output. I've moved them in the `docs-src` directory only.

Comment (robscott) [reply docs/simple-gateway/index.html:552]: It looks like this was moved in docs-src as well though?
Screen Shot 2020-12-03 at 12 04 30 PM

Comment (hbagdi) [reply docs/simple-gateway/index.html]: My bad. Updated. Thanks for catching this.

**Comment by robscott** [comment]
Thanks!

/lgtm

**Pull request merged**